sql-plugin-api/src/main/scala/com/nvidia/spark/rapids/ShimLoader.scala

```diff
@@ -57,8 +56,9 @@ import org.apache.spark.util.MutableURLClassLoader
     Using these Jar URL's allows referencing different bytecode produced from identical sources
     by incompatible Scala / Spark dependencies.
  */
-object ShimLoader extends Logging {
-  logDebug(s"ShimLoader object instance: $this loaded by ${getClass.getClassLoader}")
+object ShimLoader {
+  val log = org.slf4j.LoggerFactory.getLogger(getClass().getName().stripSuffix("$"))
```

Comment:
So much better than what we were previously doing! I don't know why we even bothered with the `Logging` trait

Reply:
It is/was a convenient pattern but it is an internal class that would now need shimming. And resulted in a chicken-and-egg problem for shim loading.